docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -206,13 +226,15 @@ pipeline {
         image: returntocorp/semgrep
         workdir: /<org_name>/<repo_name>
         environment:
-          - "SEMGREP_RULES=p/security-audit p/secrets" # more at semgrep.dev/explore
+          # Scan with rules set in Semgrep App's rule board
+          # Make a token at semgrep.dev/orgs/-/settings/tokens
+          - "SEMGREP_APP_TOKEN=${SEMGREP_APP_TOKEN}"
 
         # == Optional settings in the `environment:` block
 
-        # Instead of `SEMGREP_RULES:`, use rules set in Semgrep App.
-        # Get your token from semgrep.dev/manage/settings.
-        #   - "SEMGREP_APP_TOKEN=${SEMGREP_APP_TOKEN}"
+        # Instead of `SEMGREP_APP_TOKEN:`, set hard-coded rulesets, 
```

Comment:
So this no longer uses rules set in Semgrep App? Maybe we should specify this information still somehow.

Reply:
Maybe I just got confused by moving stuff around. Feel free to ignore this and the comment above if it is so. :-)

Reply:
yeah, see line 231. We're still using the Semgrep App, just by default now, rather than hard-coded by default.